ADOPTERS.md

```diff
@@ -0,0 +1,15 @@
+ADOPTERS
+========
+
+PUBLIC REFERENCES
+-----------------
+
+This document lists the organizations that use Idpbuilder
+If any organization would like to get added or removed please make a pull request by following these guidelines:
+
+*  For the organization column make the organization name a link that points to the orgs website if available. Please don't include your organization's logo or other trademarked material.
```

Comment:
```suggestion
* For the organization column make the organization name a link that points to the org website if available. Please don't include your organization's logo or other trademarked material.
```

Reply:
this looks good apply the suggested changes